js/manageGlossModal.js

```diff
@@ -85,7 +85,7 @@ class ManageGlossModal extends HTMLElement {
             const glossTitle = `${published ? "✓" : "❌"}  ${glossData.title}`
 
             const removeBtn = `<input type="button" value="delete" glossid="${glossID}" data-type="named-gloss" class="removeCollectionItem button error is-small" title="Delete This Entry">`
-            const visibilityBtn = `<input type="button" value="${published ? "unpublish" : "publish"}" class="togglePublic button ${published ? "error" : "success"} is-small" glossid="${glossID}" title="Toggle public visibility"/>`
+            const visibilityBtn = `<input type="button" value="${published ? "unpublish" : "publish"}" class="togglePublic button ${published ? "error" : "success"} is-small" glossid="${glossID.replace(/^https?:/, 'http:')}" title="Toggle public visibility"/>`
```

Comment:
This trick should be extracted soon, if it is needed like this. It may be in utils already

Reply:
There is some real funny stuff going on with manage glosses.  We need to go through this again.  See #179